tests/functional/ovndbcluster_controller_test.go

```diff
@@ -226,6 +227,51 @@ var _ = Describe("OVNDBCluster controller", func() {
 			DeferCleanup(th.DeleteInstance, instance)
 		})
 
+		It("should create services", func() {
+			internalAPINADName := types.NamespacedName{Namespace: namespace, Name: "internalapi"}
+			nad := th.CreateNetworkAttachmentDefinition(internalAPINADName)
+			DeferCleanup(th.DeleteInstance, nad)
+
+			statefulSetName := types.NamespacedName{
+				Namespace: namespace,
+				Name:      "ovsdbserver-sb",
+			}
+			th.SimulateStatefulSetReplicaReadyWithPods(
+				statefulSetName,
+				map[string][]string{namespace + "/internalapi": {"10.0.0.1"}},
+			)
+			service := types.NamespacedName{
```

Comment:
namespacename object doesn't look necessary, can just pass the namespace as that's what used by GetServicesListWithLabel.

Reply:
Done